src/Estimators/include/BipedalLocomotion/ContactDetectors/ContactDetector.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file ContactDetector.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_CONTACT_DETECTOR_H
+#define BIPEDAL_LOCOMOTION_CONTACT_DETECTOR_H
+
+#include <BipedalLocomotion/System/Advanceable.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/Contacts/Contact.h>
+
+#include <iostream>
+#include <unordered_map>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+
+using ContactStates = std::unordered_map<std::string, BipedalLocomotion::Contacts::EstimatedContact>;
```

Comment:
I would say that ``ContactStates`` seems a different thing from``EstimatedContact``. ``EstimatedContactList``?

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/142/commits/1e57e4c40ed9bff020e6ad045ab4b66c76b1aaa3